src/main/java/org/killbill/billing/client/KillBillClient.java

```diff
@@ -461,6 +461,12 @@ public void cancelSubscription(final UUID subscriptionId, @Nullable final Entitl
 
     public void cancelSubscription(final UUID subscriptionId, @Nullable final DateTime requestedDate, @Nullable final EntitlementActionPolicy entitlementPolicy, @Nullable final BillingActionPolicy billingPolicy,
                                    final int timeoutSec, final String createdBy, final String reason, final String comment) throws KillBillClientException {
+
+        cancelSubscription(subscriptionId, requestedDate, null, entitlementPolicy, billingPolicy, timeoutSec, createdBy, reason, comment);
+    }
+
+    public void cancelSubscription(final UUID subscriptionId, @Nullable final DateTime requestedDate, @Nullable final Boolean useRequestedDateForBilling, @Nullable final EntitlementActionPolicy entitlementPolicy,
+                                   @Nullable final BillingActionPolicy billingPolicy, final int timeoutSec,  final String createdBy, final String reason, final String comment) throws KillBillClientException {
         final String uri = JaxrsResource.SUBSCRIPTIONS_PATH + "/" + subscriptionId;
```

Comment:
@sruthipendyala Since the query parameter is a boolean, i suggest you pass a boolean in your method instaed of a string. Can you make the change (and then we 'll merge the PR) ?